google/cloud/aiplatform/metadata/types/google_types.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import Optional, Dict, NamedTuple, List
+from dataclasses import dataclass
+from google.cloud.aiplatform.metadata.types import base
+from google.cloud.aiplatform.metadata.types import utils
+
+
+class VertexDataset(base.BaseArtifactSchema):
+    """An artifact representing a Vertex Dataset."""
```

Comment:
I think we probably should consider add the version info in the comments as well in case we need to update the class and bump up the version in the future.

Reply:
the version parameter already covers this with the following 
``` 
       schema_version (str):
            Optional. schema_version specifies the version used by the Artifact.
            If not set, defaults to use the latest version.
```
kindly let me if this is sufficient.

Reply:
What I meant was that the class definition was for version 0.0.1 only, and a different version would like not able to use this class directly(0.0.2 would change the dataset name to required), so to a certain degree the class would already implicitly set the version, and if so, we might just add the version info in the class comment. I don't feel very strongly about this though.